src/config/rds_aurora.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Aurora RDS",
+  "displayName": "Amazon Aurora RDS",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "/aws/rds/cluster/%{DATA:resource_id}/%{GREEDYDATA:log_type}"
+      },
+      "sources": [
+        {
+          "sourceType": "logs",
+          "source": "log_group",
+          "condition": "$prefix('/aws/rds/cluster/')"
+        }
+      ],
+      "attributes": [
+        {
+          "key": "aws.service",
+          "pattern": "'rds'"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "log_stream",
+          "pattern": "if(log_type == 'postgresql', &replace_regex(log_stream, '\\.0$', ''), &log_stream, @ )",
+          "priority": 9
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, \"log_stream\"])",
+          "priority": 10
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "dt_meid_rds_v2(\"aws.arn\")"
+        },
+        {
+          "key": "severity",
+          "pattern": "if(log_type == 'slowquery', &'WARNING', &null, @)",
+          "priority": 11
+        },
+        {
+          "key": "severity",
+          "pattern": "if(log_type == 'postgresql', &if(contains(log_content, ':WARNING:'), &'WARNING', &if(contains(log_content, ':ERROR:'), &'ERROR', &'INFO', @), @), &if(contains(log_content, '[Warning]'), &'WARNING', &if(contains(log_content, '[Error]'), &'ERROR', &null, @), @), @)",
```

Comment:
Can we split this entry into 2? One with if postgresql then something (otherwise null), second one if mysql then something (otherwise null).
And then 3d one if severity null then severity=INFO

Reply:
If was before but @mswiatkowska found another use case that it can't

Reply:
If we wanted to have 3rd separate 'if' we would need to add additional check for if not slowquery and not postgresql -> it's too much really :D

Reply:
I really think about removing this severity conditions and send INFO for all rds logs...

Reply:
I don't get it, 3rd would be if mysql then ...

Reply:
Ok, I get it now. I wish json had comments so we could add this info

Reply:
You can always do it like that - I add description/whatever field that will get discarded when rules are loaded into class -  imho worth it for confounded ones.
```
        {
          "key": "audit.result",
          "description": "If a field errorCode exists, then return Failure.{errorCode}, else return Succeeded",
          "pattern": "if( log_content.errorCode==null, &'Succeeded', &[`Failed`, log_content.errorCode] | [? @!=null ] | join('.', @) , @)"
        },
```

Reply:
@dt-be it was already merge before I saw your comment, but will prepare another PR with that description